src/index.html

```diff
@@ -22,6 +22,347 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a
+        class="logo"
+        href="index.html"
```

Comment:
For the `href` attribute of the logo link, it's common to use `/` (root of the site) instead of `index.html`. This ensures that the link will always direct to the root index file, even if the structure of the URLs changes.